test/transcoding/SPV_INTEL_token_type/token_type_intel_negative.ll

```diff
@@ -0,0 +1,37 @@
+;; Check whether the translator reports an error for a module with token type
+;; if SPV_INTEL_token_type extension is not used.
+
+; RUN: llvm-as < %s -o %t.bc
+; RUN: not llvm-spirv %t.bc 2>&1 --spirv-allow-unknown-intrinsics | FileCheck %s
+
+; CHECK: RequiresExtension: Feature requires following SPIR-V extension:
```

Comment:
I think it is also good to say which exact feature requires extension. i.e. something like this:
```
RequiresExtension: Feature:
declare token @llvm.tokenfoo()
requires following SPIR-V extension:
SPV_INTEL_token_type

Reply:
Kind of applied, guess it's the best, what I can do here

Reply:
That is good enough for me, thanks!